test/test_layout.cpp

```diff
@@ -16,16 +16,16 @@
 
 #include "doctest/doctest.h"
 
-#include "sparrow/layout.hpp"
+#include "sparrow/fixed_size_layout.hpp"
 
 namespace sparrow
 {
     // TODO: Test all the other base types once #15 is addressed.
-    static_assert(std::ranges::contiguous_range<primitive_layout<uint8_t>>);
-    static_assert(std::contiguous_iterator<primitive_layout_iterator<uint8_t, true>>);
-    static_assert(std::contiguous_iterator<primitive_layout_iterator<uint8_t, false>>);
+    // static_assert(std::ranges::contiguous_range<fixed_size_layout<uint8_t>>);
+    // static_assert(std::contiguous_iterator<fixed_size_layout_iterator<uint8_t, true>>);
+    // static_assert(std::contiguous_iterator<fixed_size_layout_iterator<uint8_t, false>>);
 
-    using layout_test_type = primitive_layout<uint8_t>;
+    using layout_test_type = fixed_size_layout<uint8_t>;
```

Comment:
I think it would be better to test the layout with `int32_t` or any primitive type different from `uint8_t`, to be sure the internal cast are correctly done everywhere.